test/metabase/api/action_test.clj

```diff
@@ -168,28 +209,34 @@
                    (:message (mt/user-http-request :crowberto :post 400 action (dissoc request-body :query))))))))))
 
 (deftest row-update-action-gives-400-when-matching-more-than-one
-  (actions.test-util/with-actions-enabled
-    (let [query-that-returns-more-than-one (assoc (mt/mbql-query users {:filter [:>= $id 1]}) :update_row {:name "new-name"})
-          result-count (count (mt/rows (qp/process-query query-that-returns-more-than-one)))]
-      (is (< 1 result-count))
-      (doseq [{:keys [action]} (filter #(= "action/row/update" (:action %)) (mock-requests))
-              :when (not= action "action/row/create")] ;; the query in create is not used to select values to act upopn.
-        (is (re= #"Sorry, this would update [\d|,]+ rows, but you can only act on 1"
-                 (:message (mt/user-http-request :crowberto :post 400 action query-that-returns-more-than-one))))
-        (is (= result-count (count (mt/rows (qp/process-query query-that-returns-more-than-one))))
-            "The result-count after a rollback must remain the same!")))))
+  (mt/test-drivers (mt/normal-drivers-with-feature :actions)
+    (actions.test-util/with-actions-enabled
+      (let [query-that-returns-more-than-one (assoc (mt/mbql-query users {:filter [:>= $id 1]})
+                                                    :update_row {(format-field-name :name) "new-name"})
+            result-count                     (count (mt/rows (qp/process-query query-that-returns-more-than-one)))]
+        (is (< 1 result-count))
+        (doseq [{:keys [action]} (filter #(= "action/row/update" (:action %)) (mock-requests))
+                :when            (not= action "action/row/create")] ;; the query in create is not used to select values to act upopn.
+          (is (schema= {:message #"Sorry, this would update [\d|,]+ rows, but you can only act on 1"
+                        s/Keyword s/Any}
+                       (mt/user-http-request :crowberto :post 400 action query-that-returns-more-than-one)))
+          (is (= result-count (count (mt/rows (qp/process-query query-that-returns-more-than-one))))
+              "The result-count after a rollback must remain the same!"))))))
```

Comment:
The rest of this stuff is mostly unchanged but I replaced

```clj
(re= <pattern> (:message <results>))
```

with 

```clj
(schema= {:message <pattern>, s/Keyword s/Any} <results>
```

because with the old form the test failure is meaningless if the response does not contain a `:message` at all. With the new form you see the entire response when it fails